### DISCUSSION TITLE
How do I to show blog posts at '/blog/` instead of '/' ?

### DISCUSSION THREAD

**Discussion by jjaimealeman** [arthelokyo/astrowind#110] [Q&A]
Currently, the posts are at site root `http://localhost:3000/get-started-website-with-astro-tailwind-css`

Where/What do I change to make it `http://localhost:3000/blog/get-started-website-with-astro-tailwind-css`

![screenshot](https://i.imgur.com/cyKvrrF.png)

I'm still getting familiar with Astro and have been going through all the files, but I can not locate the correct place to make this change 🤔

**Comment by jjaimealeman** [comment]
Also this. How can I make it go to `http://localhost:3000/blog/page/2` instead of `http://localhost:3000/blog/2`.
![screenshot](https://i.imgur.com/SIBRBdv.png)

**Comment by prototypa** [comment]
Hi @jjaimealeman

**1 - First question:**

**AstroWind** has a setting for it in the `src/config.mjs` (https://github.com/onwidget/astrowind/blob/main/src/config.mjs#L35). It's called _permalink_.

If you change _permalink_ to `/blog/%slug%` it will work as you expect.

**2 - Second question:**

For this we use the `paginate` function that **Astro** provides and the default behavior is what you see.

- https://docs.astro.build/en/reference/api-reference/#paginate
- https://github.com/withastro/astro/blob/main/packages/astro/src/core/render/paginate.ts

As I see in the code of the function there is no easy way to add some fragment to the route. It would have to be done by playing with the routes and conditions. Example, add a new route within the blog: `[...blog]/page/[...page].astro` with the same code of the page `[...blog]/[...page].astro` and put a condition that does not generate anything if it is the first page and also make a change to blog/[...page].astro that does not paginate, only shows the first page.

**Reply by jjaimealeman** [reply]
You are _too amazing!_  thank you 👍
